### DISCUSSION TITLE
다른 플레이어의 점프 모션이 보이지 않습니다.

### DISCUSSION THREAD

**Discussion by capgreen02** [naverz/zepeto-studio-kor#815] [5. Q&A]
https://user-images.githubusercontent.com/98267406/170710786-3e0ef966-ece1-4d48-8c99-465ee6e01b39.mp4

publish 이후 테스트링크를 통해 접속했습니다.
지금 다른 플레이어가 점프를 하고 있음에도 점프하는 모션이 보이지 않습니다.
유니티에서 테스트할 때 역시 점프 모션이 보이지 않았습니다. 혹시 제페토의 문제인가요? 아니면 어디서 코드가 잘못된건가요?

**Comment by DrunkenPoy** [comment]
서버에서 전달받은 State값을 기준으로 Jump()를 실행해주고 있다면 V1인지 v2인지에 따라 점프 스테이트가 다릅니다.


코드가 어딘가 잘못되어 있을 가능성이 높으신 것 같습니다.

https://www.mettit.com/pt/view/PRE20220527110346437 

해당 글 참고해보시면 도움 되실 수도 있을 것 같습니다.
